playbooks/roles/deploy-osh/files/010-setup-client.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2017 The Openstack-Helm Authors.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+set -xe
+
+sudo -H mkdir -p /etc/openstack
+sudo -H chown -R $(id -un): /etc/openstack
+tee /etc/openstack/clouds.yaml << EOF
+clouds:
+  openstack_helm:
+    region_name: RegionOne
+    identity_api_version: 3
+    auth:
+      username: 'admin'
+      password: 'password'
+      project_name: 'admin'
+      project_domain_name: 'default'
+      user_domain_name: 'default'
+      auth_url: 'http://keystone.openstack.svc.cluster.local/v3'
+EOF
+
+#NOTE: Build charts
+make all
```

Comment:
I would prefer if we make all the charts explicitly, rather than in here. We are building the osh-infra charts in main anyway, it would be just nice to have the osh charts built afterwards, and more explicit. The airship squad could add their repos too.

Reply:
That's more a nit than a problem for me, if we keep the file. Just saying :)

Reply:
OK